src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/ElasticsearchCaseSearchOperation.java

```diff
@@ -49,37 +55,60 @@ public ElasticsearchCaseSearchOperation(JestClient jestClient,
     }
 
     @Override
-    public CaseSearchResult execute(CaseSearchRequest caseSearchRequest) {
-        SearchResult result = search(caseSearchRequest);
+    public CaseSearchResult execute(CrossCaseTypeSearchRequest request) {
+        MultiSearchResult result = search(request);
         if (result.isSucceeded()) {
             return toCaseDetailsSearchResult(result);
         } else {
             throw new BadSearchRequest(result.getErrorMessage());
         }
     }
 
-    private SearchResult search(CaseSearchRequest caseSearchRequest) {
-        Search searchRequest = secureAndTransformSearchRequest(caseSearchRequest);
+    private MultiSearchResult search(CrossCaseTypeSearchRequest request) {
+        MultiSearch multiSearchAction = secureAndTransformSearchRequest(request);
```

Comment:
can we rename from multiSearchAction to multiSearch?

Reply:
done